src/services/api.js

```diff
@@ -0,0 +1,22 @@
+const API_URL = 'https://bootcamp-api.codeit.kr/api';
+
+export async function getFolder() {
+  try {
+    const response = await fetch(`${API_URL}/sample/folder`);
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    console.log(error);
+    throw new Error('통신 오류');
+  }
+}
+export async function getUser() {
+  try {
+    const response = await fetch(`${API_URL}/sample/user`);
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    console.log(error);
+    throw new Error('통신 오류');
+  }
+}
```

Comment:
💊 
전체적으로 코드가 너무 잘 작성 되었습니다!
몇 가지 개선 할 수 있는 포인트는
1. 에러 처리 개선
2. 중복 코드를 하나의 로직에서 처리 정도가 될 것 같습니다!

아래는 제가 최대한 비슷하게 수정을 해본 코드입니다~!

```suggestion
const API_URL = 'https://bootcamp-api.codeit.kr/api';

async function fetchData(endpoint) {
  try {
    const response = await fetch(`${API_URL}${endpoint}`);
    if (!response.ok) {
      throw new Error(`Server responded with ${response.status}: ${response.statusText}`);
    }
    const result = await response.json();
    return result;
  } catch (error) {
    console.error(error);
    throw new Error('Network error');
  }
}

export function getFolder() {
  return fetchData('/sample/folder');
}

export function getUser() {
  return fetchData('/sample/user');
}
```